tempesta_fw/http_msg.c

```diff
@@ -1443,21 +1443,11 @@ tfw_http_msg_insert(TfwMsgIter *it, char **off, const TfwStr *data)
 	int r;
 	TfwStr dst = {};
 
-	if ((r = ss_skb_get_room(it->skb_head, it->skb, *off, data->len, &dst)))
+	if ((r = ss_skb_get_room_w_frag(it->skb_head, it->skb, *off, data->len, &dst, &it->frag))) {
```

Comment:
Good choice to pass the exact modified fragment to the caller! 

It seems like `skb_head`, `skb`, `frag` parameters  represent `TfwMsgIter` structure, and in most places where the `get_room()` is called right in a context where an iterator is defined and used to provide values for `get_room()`. But I bet, most functions doesn't expect that the iterator may change after `get_room()` call. Same I can say about all the similar functions in this PR.

Reply:
Not all users of the function `ss_skb_get_room...()` use `TfwMsgIter`, besides, I did not want to introduce the dependency ss_skb.h on msg.h